x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/email/email_params.test.tsx

```diff
@@ -17,6 +17,7 @@ describe('EmailParamsFields renders', () => {
       subject: 'test',
       message: 'test message',
     };
+
```

Comment:
nit: This new space can be removed and than the file won't be affected by the changes

Reply:
The `eslint` puts the end-of-file line for me when I am saving a file.